docs/guide/deeplinking.md

```diff
@@ -0,0 +1,7 @@
+# Deeplinking
+
+[Create deeplinks for your application here](https://metamask.github.io/metamask-deeplinks/#)
+
+Deeplinks allows for instant invocation of the user's preferred wallet application (even if it is a webapp or a swarm đapp), with the correct parameterization of the transaction that can then only be confirmed by the (authenticated) user.
+
+URLs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications.
```

Comment:
Add main use-cases here to help devs imagine when and why they could use deep-links. 

Suggestion:
- Create a link so your users can open your dapp directly in MetaMask app and interact with your dapp with their web3 account
- Provide a simple one-click experience such that users can easily make payments to another account (with pre-filled parameters like recipient address, amount, network, etc.)
- Does your dapp have a Connext payment channel open? Let your users make gasless and instant transactions with your dapp with Payment Channel requests! (note, this is subject to work only if the user has opted in for InstaPay experimental feature)